src/utils/sanitizeText.js

```diff
@@ -0,0 +1,6 @@
+export function sanitizeText(text) {
```

Comment:
Can merge formateDate.js and sanitizeText.js into one file and named it utilities.js ?
https://github.com/rsa-wtag/Todos/pull/1/files/5b56cec73281cf67c2ac3db38019c4e892ebce08#r1201489899

Reply:
@rsa-wtag vai, according to Sujon vai - as text sanitization and date formatting are not related tasks, they should be in separate files.